src/app/GlobalStyles.tsx

```diff
@@ -6,16 +6,19 @@ const GlobalStyle = createGlobalStyle`
 }
 
 :root {
-  --background-title: #000fff;
+  --background-primary: #000fff;
+  --background-secondary: white;
+  --font-primary: white;
```

Comment:
I would add `color` to make sure, these variables only contain colors.
A `background` could be `linear-gradient` or `url` too.
`font` could be `font-family`.

```suggestion
  --color-background-primary: #000fff;
  --color-background-secondary: white;
  --color-primary: white;
```